diff --git a/node_modules/react-native-flipper/react-native.config.js b/node_modules/react-native-flipper/react-native.config.js
index 9b962ed..b31a07c 100644
--- a/node_modules/react-native-flipper/react-native.config.js
+++ b/node_modules/react-native-flipper/react-native.config.js
@@ -8,13 +8,13 @@
  */
 
 module.exports = {
-  dependency: {
-    platforms: {
-      ios: {
-        // This file should not be needed at all, but without it.
-        // cli 4.2.2 will search for an xcode project
-        project: '.',
-      },
-    },
-  },
+  // dependency: {
+  //   platforms: {
+  //     ios: {
+  //       // This file should not be needed at all, but without it.
+  //       // cli 4.2.2 will search for an xcode project
+  //       project: '.',
+  //     },
+  //   },
+  // },
 };
